x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -308,6 +308,13 @@ export const ADD_TAG_CUSTOM_OPTION_LABEL = (searchValue: string) =>
     values: { searchValue },
   });
 
+export const COMMENT_VERSION_CONFLICT_WARNING = i18n.translate(
+  'xpack.cases.configure.commentVersionConflictWarning',
+  {
+    defaultMessage: 'This case has been updated. Please refresh before saving additional updates.',
```

Comment:
Just double checking but we only show this when the _comment_ has a version conflict right?

How about we change the message to something like: `This comment was updated. Saving your changes will overwrite the updated value.`

Reply:
What about `Comment updated. Please refresh to edit.`? cc @lcawl

Reply:
we are not stopping user to continue editing. So I think this comment is more clear: `This comment was updated. Saving your changes will overwrite the updated value.`

Reply:
> Just double checking but we only show this when the _comment_ has a version conflict right?

yes

Reply:
Oh, I thought we were preventing the users from saving the comment. Should we do that?

Reply:
yeah, we can prevent. In that case, we show message as a warning and disable save button?

Reply:
Let's discuss it offline. I am not sure what is best 🙂

Reply:
Updated the warning message: 
![image](https://user-images.githubusercontent.com/117571355/206231428-1514f238-e63f-46d0-9e92-3f955f0c110b.png)